objects/obj_controller/Create_0.gml

```diff
@@ -936,7 +936,12 @@ trade_mnum[4]=0;
 // ** Sets up starting requisition **
 requisition=500;
 if (instance_exists(obj_ini)){
-    if (obj_ini.progenitor==0) /*and (obj_creation.custom=0)*/ and (global.chapter_name!="Doom Benefactors") then requisition=2000;
+    if (
+        (obj_ini.progenitor == PROGENITOR.DARK_ANGELS) &&
+        (global.chapter_name != "Doom Benefactors")
+    ) {
+        requisition=2000;
+    }
```

Comment:
I think in here it meant "if you don't have a progenitor - get 2000 requisition". I don't remember if 0 was none or random. I think none, right?

Reply:
This is changed to `PROGENITOR.NONE` in [184fe4f](https://github.com/EttyKitty/ChapterMaster/pull/44/commits/184fe4fcf3b2304a2d8baedc0847da80dd1509fc)